src/main/java/org/jeasy/props/processors/AbstractAnnotationProcessor.java

```diff
@@ -1,25 +1,25 @@
-/**
```

Comment:
Please leave these license header changes out of the PR. I use a [template](https://github.com/j-easy/easy-props/blob/master/licence-header-template.txt) with the [maven licence plugin](https://github.com/j-easy/easy-props/blob/master/pom.xml#L175) to update these headers. So if there is an issue with the template, we need to change the template itself so that the changes are taken into account to all other files.